test/org/sagebionetworks/bridge/models/accounts/PhoneTest.java

```diff
@@ -0,0 +1,38 @@
+package org.sagebionetworks.bridge.models.accounts;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+import org.sagebionetworks.bridge.json.BridgeObjectMapper;
+
+import com.fasterxml.jackson.databind.JsonNode;
+
+public class PhoneTest {
+
+    @Test
+    public void canSerialize() throws Exception {
+        Phone phone = new Phone("408.258.8569", "US");
+        assertEquals("(408) 258-8569", phone.getNationalFormat());
```

Comment:
We should have separate explicit tests for getNationalFormat() and getNumber() instead of merging them with JSON serialization tests. As is, we only indirectly verify getNumber() returns the data we expect it to.